MOBILE_CLIENT/ANDROID/_LOW/APK_DEBUG_SYMBOL_PRESENT/recommendation.md

```diff
@@ -0,0 +1 @@
+Remove all symbols and debug data from the application. To do so, configure the build type to exclude debug information.
```

Comment:
Missing how to remove them. There are command line tools to strip them, and compiler flags to remove them. etc.